include/seqan3/alignment/matrix/alignment_coordinate.hpp

```diff
@@ -77,8 +77,10 @@ class advanceable_alignment_coordinate
 {
 public:
 
-    //!\name Member types
-    //!\{
+    /*!\name Member types
+     * \{
+     */
```

Comment:
does this trigger warnings in doxygen?

Reply:
Sadly, yes. It has to be a block comment, otherwise the following code remains undocumented:
```
seqan3/alignment/matrix/alignment_coordinate.hpp:83: warning: Member difference_type (typedef) of class seqan3::detail::advanceable_alignment_coordinate is not documented.
```

Reply:
😭